userpanel/style/bclean/style.css

```diff
@@ -32,6 +32,10 @@ TEXTAREA.alert:focus {
 
 .text-center	{ text-align: center; }
 
+.lms-userpanel-userassignment-head, .lms-userpanel-userdocuments-head, .lms-userpanel-usernodes-head, .lms-userpanel-userbalance-head, .lms-userpanel-helpdesklist-head {
```

Comment:
moim zdaniem powinna być klasa `lms-userpanel-table-header`, bo po co tyle oddzielnych klas? Będziemy chcieli sterować wyglądem każdej tabeli oddzielnie? Wątpię w to.

Reply:
W drugą stronę to nie działa, a nie będziemy sobie chcieć ikonek popakować w nagłówki za pomocą CSS?

Reply:
Ikonki to pewnie będą oddzielne klasy.

Reply:
No dobra ale Ty chcesz je pakować do kodu szablonu a nie lepiej żeby były podklasami w kodzie takiej klasy?

Reply:
Czyli:
- nazywamy miejsce wg lokalizacji fizycznej,
- ewentualnie łączymy w grupy o podobnych właściwościach
- cała magia dzieje się w CSS
(dzięki temu będzie też można je wyrzucić łatwo w przypadku gdy ostylujemy sobie widok na mały rozmiar)

Reply:
Przecież jeden element interfejsu może mieć wiele klas - jedna klasa odpowiedzialna z ogólny wygląda, druga klasa z ikonkę.

Reply:
Ja pytam o to czy lepiej:
a) każdy element był w CSS był w szablonie HTML opisany **tylko jedną** klasą a reszta w CSS,
b) każdy element ma wiele klas w szablonie HTML i opisany jest wiloma podklasami CSS,
?

Reply:
często robi się przez kilka klas - zresztą spójrz nawet na bootstrap. Jedna klasa może dawać jedno zachowanie, druga inne.

Reply:
Ok poprawię

Reply:
Poprawione

Reply:
@chilek będzie ok?